### DISCUSSION TITLE
4.x TreeSelect 输入框中Tag的展示和3.x的不一致

### DISCUSSION THREAD

**Discussion by Lpbzzz** [ant-design/ant-design#37129] [Q&A] [answered]
4.x的tag会隐藏全选的tree父级节点，3.x的不会
<img width="996" alt="image" src="https://user-images.githubusercontent.com/28721487/185282349-726d9a65-7a74-466b-8058-d41f49671795.png">
<img width="950" alt="image" src="https://user-images.githubusercontent.com/28721487/185282503-677265c4-b1bc-4085-b3e3-5caf565ca85d.png">

**Comment by zombieJ** [comment, marked as answer]
`showCheckedStrategy`  可以设置展示全部
